<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>113</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>113</y>
     <width>1920</width>
     <height>967</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8e8e8; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>14</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>147</y>
     <width>1800</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border: 2px solid #ffffff;
    border-radius: 30px; /* Bordes redondeados */
    padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>36</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>E-MOVILIDAD (ADMIN)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1836</x>
     <y>34</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ola K채llenius.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAdmin">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>30</y>
     <width>199</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #000000;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;

color: black;
background-color: #ebebeb;</string>
   </property>
   <property name="text">
    <string>Administraci처n</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>1617</x>
     <y>41</y>
     <width>209</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Ola K채llenius</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="botonRegistro">
   <property name="geometry">
    <rect>
     <x>701</x>
     <y>347</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #000000;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;

color: white;
background-color: #002d6b;</string>
   </property>
   <property name="text">
    <string>Registro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonDB">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>347</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #000000;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;

color: white;
background-color: #002d6b;</string>
   </property>
   <property name="text">
    <string>Base de Datos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonUsuarioAutorizados">
   <property name="geometry">
    <rect>
     <x>1294</x>
     <y>347</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #000000;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;

color: white;
background-color: #002d6b;</string>
   </property>
   <property name="text">
    <string>Usuarios Autorizados</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>783</x>
     <y>222</y>
     <width>354</width>
     <height>67</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>E-MOVILIDAD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="botonLogOut">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>30</y>
     <width>199</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #000000;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;

color: black;
background-color: #ebebeb;</string>
   </property>
   <property name="text">
    <string>Cerrar Sesi처n</string>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_12</zorder>
  <zorder>label_8</zorder>
  <zorder>botonAdmin</zorder>
  <zorder>label_23</zorder>
  <zorder>botonRegistro</zorder>
  <zorder>botonDB</zorder>
  <zorder>botonUsuarioAutorizados</zorder>
  <zorder>label_13</zorder>
  <zorder>botonLogOut</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
